### PR TITLE
Feature: Update the Nigerian Pizza description

### LINKED ISSUE #77: Feature: Update Nigerian Pizza description
### Current behavior

Updated description of Nigerian Pizza to highlight its fusion of culinary traditions and diverse flavors

### Suggested solution

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by madefromjames**
## Description

This PR adds the feature of enhancing Nigeria's pizza recipe.

## What type of PR is this? (check all applicable)

- [ ] ☝️ Add a pizza fact or trivia
- [x] 🧑‍🍳 Add a pizza recipe
- [ ] 🗺️ Add a regional pizza
- [ ] 📝 Documentation Update

## Related Tickets & Documents

Closes #77 

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## [optional] What GIF best describes this PR or how it makes you feel?
Partying Face

### CODE REVIEW

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @madefromjames,

Thank you for your contribution! ✨ 

I have a little review here for you to fix.

Also, please address below things:

- In the "Related Tickets and Documents" section, please link to your issue number, which is #77. And add word "Closes" before the issue number. :)
- Change the PR title to be more descriptive to your changes. For example, "Update the Nigerian Pizza description".

Please let me know if you have any question.

Thank you! 😊

**Comment by madefromjames** [comment]
> Hey @madefromjames,
> 
> Thank you for your contribution! ✨
> 
> I have a little review here for you to fix.
> 
> Also, please address the things below:
> 
> * Change the PR title to be **more descriptive to your changes**. For example, "Feature: Update the Nigerian Pizza description".
> * In the "Related Tickets and Documents" section, please **link to your issue number**, which is [Add a pizza Recipe! (Please Keep This Issue Open) #77](https://github.com/open-sauced/pizza-verse/issues/77). You can see the issue number right beside your issue title, as in the screenshot below. And please add the word "Closes" before the issue number.
>   Screenshot 2024-05-02 090457
> 
> Please let me know if you have any questions.
> 
> Thank you! 😊

Hi @adiati98 I appreciate your follow-up with me, I have linked the issue number.
image
I have also committed changes to the header `Nigeria Pizza`
Please let me know what more needs to be done.

File: package-lock.json
Comment (adiati98) [inline package-lock.json:1]: Please revert every change that you made in this file.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @madefromjames,

We don't want to modify any other files than the `pizza-recipes.md` here.

I have a review here for you to fix.

Please let me know when you're done. Thank you. 🙂

File: package-lock.json
Comment (madefromjames) [reply package-lock.json:1]: Okay @adiati98 I'll do that now

Comment (madefromjames) [reply package-lock.json:1]: > Please revert every change that you made in this file.

I think I have successfully revert the changes. Please let me know if everything is fine now.

**Review by adiati98 — APPROVED** [review]
Thank you, @madefromjames!

thanks gif

**Review by BekahHW — APPROVED** [review]

**Pull request merged**